doc/src/site/sphinx/configuration/configuration_properties.rst

```diff
@@ -168,13 +168,13 @@ Configuration properties independent of selected backend
 |                                                    |                |                                                 | if the property |secureConnections| is |
 |                                                    |                |                                                 | set to ``true``.                       |
 +----------------------------------------------------+----------------+-------------------------------------------------+----------------------------------------+
-| ``spark.ext.h2o.hive.enabled``                     | ``false``      | ``setHiveSupportEnabled()``                     | If enabled, H2O instances will create  |
-|                                                    |                |                                                 | JDBC connections to Hive so that H2O   |
-|                                                    |                | ``setHiveSupportDisabled()``                    | Python & R API will be able to read    |
-|                                                    |                |                                                 | data from HiveServer2. Don't forget to |
-|                                                    |                |                                                 | put a jar with Hive driver on Spark    |
-|                                                    |                |                                                 | classpath if the internal backend is   |
-|                                                    |                |                                                 | used.                                  |
+| ``spark.ext.h2o.kerberized.hive.enabled``          | ``false``      | ``setKerberizedHiveEnabled()``                  | If enabled, H2O instances will create  |
```

Comment:
IMHO, this part is not accurate anymore :
`so that H2O Python & R API will be able to read data from HiveServer2.` If I'm not mistaken, we can use Scala and SW api to do that.

Reply:
Sorry, missed this comment. Will fix